verilog/analysis/checkers/constraint_name_style_rule.h

```diff
@@ -15,6 +15,8 @@
 #ifndef VERIBLE_VERILOG_ANALYSIS_CHECKERS_CONSTRAINT_NAME_STYLE_RULE_H_
 #define VERIBLE_VERILOG_ANALYSIS_CHECKERS_CONSTRAINT_NAME_STYLE_RULE_H_
 
+#include <optional>
+#include <regex>
```

Comment:
Make sure to enable exceptions for this library (example in [lint_waiver]( https://github.com/google/verible/blob/7c994b1af0ef6c83439d00186abdb0c36c2320ef/common/analysis/BUILD#L99)).

We usually prefer to disable exceptions for everything _except_ for the files that absolutely need them.